docs/Ecosystem.md

```diff
@@ -148,3 +148,4 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`i18next-http-middleware`](https://github.com/i18next/i18next-http-middleware#fastify-usage) An [i18next](https://www.i18next.com) based i18n (internationalization) middleware to be used with Node.js web frameworks like express or `Fastify` and also for Deno.
 - [`k-fastify-gateway`](https://github.com/jkyberneees/fastify-gateway) API Gateway plugin for `fastify`, a low footprint implementation that uses the `fastify-reply-from` HTTP proxy library.
 - [`openapi-validator-middleware`](https://github.com/PayU/openapi-validator-middleware#fastify) Swagger and OpenAPI 3.0 spec-based request validation middleware that supports `fastify`.
+- [`fastify-autoroutes`](https://github.com/GiovanniCardamone/fastify-autoroutes) Map directory structure to routes url
```

Comment:
The ecosystem plugins is sorted by name, could change it?

Reply:
Dot is missing at the end of the sentence. Also could you make description a bit more consistent with the others? Most of them are "X is a plugin/middleware for YZ"

Reply:
yes sorry, i will change it